templates/post_type_extended.mustache

```diff
@@ -43,3 +43,32 @@ function {{machine_name}}_updated_messages( $messages ) {
 	return $messages;
 }
 add_filter( 'post_updated_messages', '{{machine_name}}_updated_messages' );
+
+/**
+ * Sets the bulk post updated messages for the `{{machine_name}}` post type.
+ *
+ * @param  array $bulk_messages Arrays of messages, each keyed by the corresponding post type. Messages are
+ *                               keyed with 'updated', 'locked', 'deleted', 'trashed', and 'untrashed'.
+ * @param  int[] $bulk_counts   Array of item counts for each message, used to build internationalized strings.
+ * @return array Bulk messages for the `{{machine_name}}` post type.
+ */
+function {{machine_name}}_bulk_updated_messages( $bulk_messages, $bulk_counts ) {
+	global $post;
+
+	$bulk_messages['{{slug}}'] = array(
+		/* translators: %s: Number of {{label_plural}}. */
+		'updated'   => _n( '%s {{label}} updated.', '%s {{label_plural}} updated.', $bulk_counts['updated'], '{{textdomain}}' ),
+		'locked'    => ( 1 === $bulk_counts['locked'] ) ? __( '1 {{label}} not updated, somebody is editing it.', '{{textdomain}}' ) :
```

Comment:
Why is this using a special case for `1` ?  Shouldn't that be covered already by the `_n()` function directly below?

Reply:
In languages like English/German, that would be the case. But in languages like Russian and similar, the numbers ending with one (21, 31, ... ) with the exception of 11 are also using the singular version of a sentence.

I've used the same messages as the core is using for the internal post types: https://github.com/WordPress/WordPress/blob/master/wp-admin/edit.php#L331